junit5/src/main/java/cz/xtf/junit5/annotations/SinceVersion.java

```diff
@@ -15,10 +15,21 @@
 @ExtendWith(SinceVersionCondition.class)
 public @interface SinceVersion {
 
+
 	/**
 	 * Name or regexp pattern matching name of the image. For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*"
+	 * <p>
+	 * Only one of {@code dockerImageName} and {@code name} can be presented.
+	 */
+	String name() default "";
+
+	/**
+	 * Name or regexp pattern matching name in {@code Docker Labels} in image metadata
+	 * For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*".
+	 * <p>
+	 * Only one of {@code dockerImageName} and {@code name} can be presented.
 	 */
-	String name();
+	String dockerImageName() default "";
```

Comment:
`dockerImageName` does directly provide info what it is. What about to rename it to something like: `ImageMetadataLabelName`, wdyt?